### ISSUE TITLE
Textbox and dropdown in Balloon panel not working in Firefox

### ISSUE THREAD

**Issue by deadpoollove134** [ckeditor/ckeditor4#3583]
## Type of report

Bug

## Provide detailed reproduction steps (if any)

1. Open CK Editor in Firefox, Click on accessibility checker icon. My content loads in Bootstrap 4 modal and validates for accessibility issues.
2. A Balloon panel appears where we have text box to give alt text for the image content.
3. The text box is not clickable and not working in Firefox browser alone.

### Expected result

*I should be able to type the alt text in the text box*

### Actual result

*The textbox is not clickable and not working*

## Other details

* Browser: Firefox
* OS: All OS
* CKEditor version: 4
* Installed CKEditor plugins: Accessibility plugin, Balloon Panel plugin

**Comment by Dumluregn** [comment]
Hello, thank you for interest in CKEditor! As for the issue, I tried to reproduce it using our [Accessibility Checker Plugin sample](https://ckeditor.com/docs/ckeditor4/latest/examples/accessibilitychecker.html) and it seems to be working fine. Could you check it too?
![a11ychecker](https://user-images.githubusercontent.com/10883184/67074157-d309ec80-f188-11e9-8ace-43b4e1784901.gif)

**Comment by Comandeer** [comment]
Hi, It's been a while since we last heard from you. We're closing this issue for now. Still, feel free provide us requested feedback, so that we can reopen this issue.
